examples/simple-game-server/main.go

```diff
@@ -693,48 +693,48 @@ func getCounterCount(s *sdk.SDK, counterName string) (string, error) {
 }
 
 // incrementCounter returns the if the Counter Count was incremented successfully (true) or not (false)
-func incrementCounter(s *sdk.SDK, counterName string, amount string) (string, error) {
+func incrementCounter(s *sdk.SDK, counterName string, amount string) error {
 	amountInt, err := strconv.ParseInt(amount, 10, 64)
 	if err != nil {
-		return "false", fmt.Errorf("could not increment Counter %s by unparseable amount %s: %s", counterName, amount, err)
+		return fmt.Errorf("could not increment Counter %s by unparseable amount %s: %s", counterName, amount, err)
 	}
 	log.Printf("Incrementing Counter %s Count by amount %d", counterName, amountInt)
-	ok, err := s.Alpha().IncrementCounter(counterName, amountInt)
+	err = s.Alpha().IncrementCounter(counterName, amountInt)
 	if err != nil {
 		log.Printf("Error incrementing Counter %s Count by amount %d: %s", counterName, amountInt, err)
-		return strconv.FormatBool(ok), err
+		return err
 	}
-	return "SUCCESS: " + strconv.FormatBool(ok) + "\n", nil
+	return nil
```

Comment:
```suggestion
	return "SUCCESS\n", nil
```

So we still have a nice response when it applies correctly.

Same for all the rest 👍🏻

Reply:
Update: I have made adjustments to the code to ensure that the return type is replicated as (string, error). However, I am uncertain about what is missing to enable the Counters and Lists. sometimes, the enabling of the CL feature does not reflect with the following steps:

0. bump simple-game-server image
1. make build-agones-sdk-binary-linux-amd64 
2. Go to agones/cmd/sdk-server/bin
3. Run ./sdk-server.linux.amd64 --local --feature-gates=CountsAndLists=true
4. In another terminal, go to agones/examples/simple-game-server
5. make build-linux-image-amd64 
6. docker run --rm --network=host simple-game-server:0.30-linux-amd64 
7. In another terminal: nc -u localhost 7654

Reply:
quick question about the preferred output for different scenarios

For a successful scenario, the preferred output would be:
	return "SUCCESS\n", nil
and for error scenario:
	return "ERROR: <along with the input that we pass>"?

Reply:
@markmandel, could you please explain why we need to have two methods, one in main.go and the other in alpha.go, instead of directly calling the function in alpha.go? Additionally, please share the preferred output for the error.

Reply:
> why we need to have two methods for C & L, one in main.go and the other in alpha.go, instead of directly calling the function in alpha.go?

The alpha.go is the actual Go SDK that everyone using Agones, with Go will import and use - so we call that within main.go - which is our example of using the Go SDK. The functions in there are there just to give use some abstraction in our example.

This output looks good!